diff --git a/node_modules/sequelize/types/lib/model.d.ts b/node_modules/sequelize/types/lib/model.d.ts
index bd900b3..4b82183 100644
--- a/node_modules/sequelize/types/lib/model.d.ts
+++ b/node_modules/sequelize/types/lib/model.d.ts
@@ -2078,6 +2078,33 @@ export abstract class Model<T = any, T2 = any> extends Hooks {
     options: IncrementDecrementOptions
   ): Promise<M>;
 
+  /**
+   * Decrements a single field.
+   */
+  public static decrement<M extends Model, K extends keyof M>(
+    this: { new (): M },
+    field: K,
+    options: IncrementDecrementOptionsWithBy
+  ): Promise<M>;
+
+  /**
+   * Decrements multiple fields by the same value.
+   */
+  public static decrement<M extends Model, K extends keyof M>(
+    this: { new (): M },
+    fields: K[],
+    options: IncrementDecrementOptionsWithBy
+  ): Promise<M>;
+
+  /**
+   * Decrements multiple fields by different values.
+   */
+  public static decrement<M extends Model, K extends keyof M>(
+    this: { new (): M },
+    fields: { [key in K]?: number },
+    options: IncrementDecrementOptions
+  ): Promise<M>;
+
   /**
    * Run a describe query on the table. The result will be return to the listener as a hash of attributes and
    * their types.
